testing/runner.ts

```diff
@@ -1,44 +1,41 @@
 #!/usr/bin/env -S deno -A
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { parse } from "../flags/mod.ts";
-import { glob, isGlob, walk } from "../fs/mod.ts";
-import { runTests } from "./mod.ts";
-const { args, cwd } = Deno;
-
-const DEFAULT_GLOBS = [
-  "**/*_test.ts",
-  "**/*_test.js",
-  "**/test.ts",
-  "**/test.js"
-];
-
-/* eslint-disable max-len */
+import {
+  WalkInfo,
+  expandGlobSync,
+  glob,
+  ExpandGlobOptions
+} from "../fs/mod.ts";
+import { isAbsolute, join } from "../fs/path/mod.ts";
+import { RunTestsOptions, runTests } from "./mod.ts";
+const { DenoError, ErrorKind, args, cwd, exit } = Deno;
+
+const DIR_GLOBS = [join("**", "?(*_)test.{js,ts}")];
+
 function showHelp(): void {
   console.log(`Deno test runner
 
 USAGE:
-  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [FILES...]
+  deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [MODULES...]
 
 OPTIONS:
-  -q, --quiet               Don't show output from test cases 
-  -f, --failfast            Stop test suite on first error
-  -e, --exclude <FILES...>  List of file names to exclude from run. If this options is 
-                            used files to match must be specified after "--". 
-  
-ARGS:
-  [FILES...]  List of file names to run. Defaults to: ${DEFAULT_GLOBS.join(
-    ","
-  )} 
-`);
-}
-/* eslint-enable max-len */
+  -q, --quiet                 Don't show output from test cases
+  -f, --failfast              Stop running tests on first error
+  -e, --exclude <MODULES...>  List of comma-separated modules to exclude
+  --allow-none                Exit with status 0 even when no test modules are
```

Comment:
:+1: IMHO this is better for CI experience. Given our own incident where there were no test files found on Windows I believe that failing in that situation is a safe default.